src/main/conf/ds-cumulus-export-default-mapping.yml

```diff
@@ -96,10 +99,48 @@ digisam:
           replacement: "https://kb-images.kb.dk/DAMJP2/$1/"
           lineBreakIsMulti: false
 
-        - source: "Horizontal Pixels"
+        - source: "Horizontal Pixels"  # COP: part of Dimension
           dest: "width_pixels"
           destType: "long"
 
-        - source: "Vertical Pixels"
+        - source: "Vertical Pixels"  # COP: part of Dimension
           dest: "height_pixels"
           destType: "long"
+
+        - source: "Record Name"  # COP: Id
+          dest: "record_name"
+
+        - source: "Materialebetegnelse"  # COP: Ressourcetype 1 (If missing: Generel materialebetegnelse below is shown)
+          dest: "material_1"
+
+        - source: "Generel materialebetegnelse"  # COP: Ressourcetype 2
+          dest: "material_2"
+
+        - source: "Opstilling"  # COP: Opstilling
+          dest: "archive" # Better tag needed
+
+        - source: "Lokalitet"  # COP: Lokalitet
+          dest: "location"
+
+        - source: "Accessionsnr"  # COP: Accessionsnummer
+          dest: "accession_number"
+
+        - source: "Keywords"  # COP: Emne
+          dest: "topic"  # or replace keyword?
+
+        - source: "Note"  # COP: Kommentar
+          dest: "note"
+
+        - source: "Person" # COP: Person (Known to change format, related to "Ophav")
+          dest: "person"
```

Comment:
Can we somehow signal that the material is _about_ the person? Maybe `subject_person`? We already have the uncontrolled `subject`, but I like the idea of being explicit about it being a person.

Reply:
rename to subject_person